backend/src/test/scala/com/softwaremill/bootzooka/test/TestHelpers.scala

```diff
@@ -2,10 +2,19 @@ package com.softwaremill.bootzooka.test
 
 import java.time.{OffsetDateTime, ZoneOffset}
 
+import com.softwaremill.bootzooka.common.crypto.{Argon2dPasswordHashing, CryptoConfig, PasswordHashing}
 import com.softwaremill.bootzooka.user.domain.User
+import com.typesafe.config.Config
 
 trait TestHelpers {
 
+  implicit val hashing: PasswordHashing = new Argon2dPasswordHashing(new CryptoConfig {
+    override def rootConfig: Config = ???
+    override lazy val iterations = 2
+    override lazy val memory = 16383
```

Comment:
Zamiast to powtarzać w testach można zrobić dane domyślne w testowym configu, a `override lazy val` tylko tam, gdzie nadpisujemy wartość domyślną.

Reply:
Done